### ISSUE TITLE
this  BBEditor... command line messages on auto save.

### ISSUE THREAD

**Issue by zonkmachine** [LMMS/lmms#2183]
@Wallacoloo 
Since commit 908591ba7dc186e942305e76463029643aa21593 the terminal is 'flooded' with messages every time the auto save function hits in.

 > :~/builds/lmms$ git bisect good
908591ba7dc186e942305e76463029643aa21593 is the first bad commit commit 908591ba7dc186e942305e76463029643aa21593 Author: Colin Wallace <wallacoloo@gmail.com>
Date:   Mon Jun 15 03:26:07 2015 +0000 
Have MainWindow and others make use of the new SubWindow class

Looks like this:
```
this  BBEditor 
this  SongEditorWindow 
this  FxMixerView 
this  ControllerRackView 
this  PianoRollWindow 
this  AutomationEditorWindow 
this  ProjectNotes 
this  BBEditor 
this  SongEditorWindow 
this  FxMixerView 
 ... etc
:~
```

**Comment by Wallacoloo** [comment]
Ah, I left a debug statement in MainWindow.cpp:693 and it seems nobody
caught it in the review process...

Thanks for tracking down the relevant commit though. I'll submit a patch
for it later today unless somebody beats me to it.

**Comment by zonkmachine** [comment]
:+1:

**Comment by Wallacoloo** [comment]
Just submitted a fix: #2184.
